codeguru/views.py

```diff
@@ -54,6 +51,16 @@ def group(request, id=None):
 
                 return redirect("group", id=new_group.id)
 
+            if CgGroup.objects.all().filter(name=form.data["name"]).exists():
+                return error(request, gettext("Group already exists. try choosing another name."))

```

Comment:
This check was moved from before the group's creation to outside `is_valid`. Does it still work as expected?

Reply:
This move was intended, `CgGroup`'s model has a check for the uniqueness of the name, thus if the name isn't unique the form won't be valid
https://github.com/codeguru-il/codeguru-website/blob/0fee6aab344081ec5b16ab99ff33b6e07db39c97/codeguru/models.py#L65